recipes/zen3geo/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "zen3geo" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/zen3geo-{{ version }}.tar.gz
+  sha256: 09e84306809d03899155510f75dff83023e301c95bb0835b7bf5c34e58c72720
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - poetry-core >=1.0.0
```

TODO wait for official `poetry=1.2.0` release that comes with `poetry-core=1.1.0`, otherwise need to figure a way to add the `conda-forge/label/poetry_dev` channel to install `poetry-core=1.1.0b3` from https://anaconda.org/conda-forge/poetry-core/files/modal/info/62cddaf59aac3c40e4501101

```suggestion
    - poetry-core >=1.1.0
```